src/Symfony/Component/Serializer/Encoder/YamlEncoder.php

```diff
@@ -16,13 +16,16 @@
 use Symfony\Component\Yaml\Parser;
 
 /**
- * Encodes YAML data.
+ * Encodes YAML/YML data.
  *
  * @author Kévin Dunglas <dunglas@gmail.com>
  */
 class YamlEncoder implements EncoderInterface, DecoderInterface
 {
-    const FORMAT = 'yaml';
+    const FORMAT = self::YAML_FORMAT;
+    const YAML_FORMAT = 'yaml';
```

Comment:
the new consts feels a bit much, if you insist it on it i'd suggest to add `public const ALT_FORMAT = 'yml'`